### DISCUSSION TITLE
Accentuation of Adjective with the Genitive Singular in ĪUS

### DISCUSSION THREAD

**Discussion by rl1234567890** [DivinumOfficium/divinum-officium#2667] [Q&A]
There are nine adjectives with the genitive singular in īus (e.g., [Allen and Greenough /Latin Grammar 113](https://dcc.dickinson.edu/grammar/latin/1st-and-2nd-declension-adjectives-genitive-%c4%abus-dative-%c4%ab)) . Since the penultimate syllable is long because of the long ī, the accent should fall on it (e.g., [AG 12](https://dcc.dickinson.edu/grammar/latin/accents)).  I noticed several examples of "wrong" accentuation for such adjectives. Should they be corrected or is it an ecclesiastical usage?

**Comment by APMarcello3** [comment]
Could you cite the examples and I will cross-reference with the printed editions?

**Comment by rl1234567890** [comment]
*tótius
Martyrologium/02-03.txt Martyrologium1955R/02-03.txt Martyrologium1960/02-03.txt
Martyrologium/02-23.txt Martyrologium1955R/02-23.txt Martyrologium1960/02-23.txt
Martyrologium/03-19.txt Martyrologium1955R/03-19.txt Martyrologium1960/03-19.txt
Martyrologium/12-07.txt Martyrologium1955R/12-07.txt Martyrologium1960/12-07.txt
Martyrologium/12-20.txt Martyrologium1955R/12-20.txt Martyrologium1960/12-20.txt
Martyrologium/12-27.txt Martyrologium1955R/12-27.txt Martyrologium1960/12-27.txt
Sancti/12-29.txt (twice)
Tempora/Quadp2-0.txt

*altérius
Commune/C1a.txt
Martyrologium/03-23.txt Martyrologium1955R/03-23.txt Martyrologium1960/03-23.txt
Martyrologium/03-24.txt Martyrologium1955R/03-24.txt Martyrologium1960/03-24.txt
Martyrologium/05-19.txt Martyrologium1955R/05-19.txt Martyrologium1960/05-19.txt
Martyrologium/06-28.txt Martyrologium1955R/06-28.txt Martyrologium1960/06-28.txt
Martyrologium/09-01.txt Martyrologium1955R/09-01.txt Martyrologium1960/09-01.txt
Martyrologium/09-10.txt Martyrologium1955R/09-10.txt Martyrologium1960/09-10.txt
Martyrologium/12-12.txt Martyrologium1955R/12-12.txt Martyrologium1960/12-12.txt

However, notice that AG has

> c. The i of the genitive ending -īus, though originally long, may be made short in verse; this occurs often in alterius and regularly in utriusque.

**Reply by ajmalton** [reply]
I looked at a modern edition of the 1962, and a late 19th, and a 1770.  I looked at the Common of Evangelists Lectio 2.  All three had altérius not ateríus.  

I'm sure the original text was not typed by Laszlo by hand, but scanned.  So the edition in question has the diacritics we show.

I think they should not be changed.

**Reply by rl1234567890** [reply]
If I may further inquire, [philosophia](https://logeion.uchicago.edu/philosophia) has the penultimate syllable  short and in Latin (but not in Greek,  φιλοσοφία, since the final alpha is long) the accent would fall on the ante-penultimate, philosóphia. However, the accent in the files is on the penultimate philosophía, like in Greek:

Martyrologium/11-04.txt Martyrologium1955R/11-04.txt Martyrologium1960/11-04.txt
Sancti/01-12.txt
Sancti/04-14q.txt Sancti/04-14.txt
Sancti/05-15.txt

Shall we observe the Greek or the Latin rule?

**Reply by ajmalton** [reply]
I haven't been able to find an edition with philosóphia.  Have you?  

I continue to think we should follow the editions, rather than the grammarians.

**Reply by APMarcello3** [reply]
I strongly concur with @ajmalton. It has been my goal to simply represent the text, to the extent possible, as found in the approved liturgical text.

**Reply by rl1234567890** [reply]
Breviario Romano Latino - Italiano by Edmundo Battisti (Imprimatur 1931) has philosophía with the "Greek" accent and altérius  However, it has totíus consistently.

**Reply by KPFC** [reply]
Four different Breviaria Romana by Pustet (imprimatur between 1914 and 1954) and one from Societas S. Joannis Evangelistae (1945) have all "philosóphia". A slightly older edition from Pustet (1912) and one from Polyglotta Vaticana (imprimatur 1935) has "philosophía".
There seems not to be any consistency but rather each publisher doing his own thing.

**Reply by APMarcello3** [reply]
If this is the case then I am simply inclined to leave our texts as they are.

**Reply by rl1234567890** [reply]
1962 Editio Typica of the Missale Romanum has **totíus**  and **altérius**. I propose to modify tótius into totíus and keep the rest unchanged.

**Reply by APMarcello3** [reply]
Placet.

**Reply by APMarcello3** [reply]
Cf. https://github.com/DivinumOfficium/divinum-officium/commit/3f13b11365d5cf28e648106b93d1e414790e9f65

**Reply by fiapps** [reply]
> 1962 Editio Typica of the Missale Romanum has **totíus** and **altérius**. I propose to modify tótius into totíus and keep the rest unchanged.

The 1961 _editio typica_ of the Breviary also uses *totíus,* though not everywhere. In the hymn at Vespers for Pontiff Confessors, it has "Tótius mundi sériem gubérnat," presumably for the sake of the meter.
